llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -662,23 +724,39 @@ Error CoverageMapping::loadFunctionRecord(
     }
     Function.pushRegion(Region, *ExecutionCount, *AltExecutionCount);
 
+    if (Region.Kind == CounterMappingRegion::ExpansionRegion) {
+      for (auto &Decision : reverse(Decisions)) {
```

Comment:
would it make sense to use something like `any_of` here?

Reply:
I have rewritten whole things in `DecisionRow`.